src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -128,9 +133,11 @@ def parse_body(self, data, decode=False):
             try:
                 res = json.loads(data)
             except ValueError as exc:
-                msg = 'Unable to load JSON data\n%s' % data
-                raise ValueError(exc)
-            return res
+                msg = 'Unable to load JSON data, err=%s, data type=%s' \
+                        % (str(exc), type(data))
+                print msg
```

Comment:
The class itself does not depend on logger and neither accept logger parameter. I can import the logging module and redirect print statement to warning stream. But I don't know how WMCore handles logging modules. A long time ago Lassi convinced me that it is much better to use print to stdout/stderr instead of logging module (due to overhead, concurrency issues, etc). But I can add it if you want, similar to how we did in LogDB.